src/lib/undux/SimpleStore.js

```diff
@@ -102,16 +102,35 @@ const initialState: State = {
   serviceWorkerUpdated: null,
 }
 
+// keeping current snapshot in private module variable
+// it could be accessed via getCurrentSnapshot
+// we need it to habe show dialog outside components/hooks
+// sometimes it's needed for example in the web-only Zoom sdk wrapper
+// we have to show 'reload app' dialog on 65391 exception
+let currentSnapshot = null
```

Comment:
I dont understand why we need this. we saw that preload throws an error why isnt it enough?

Reply:
we have to catch this error & show dialog from ZoomSDK.web class as it's a web-related issue only. Architecturally we shouldn't to add this case to the hook, as hooks are. made platform-independent

But sdk class isn't a React component / hook so we couldn't call useDialog/useStore from it. We have to get current store snapshot somwhere and call showDialog passing current store snapshot to it

Architecturally it's normally as we don't performing any direct hacking manipulations. we just subscribing to store changes and could as for current state
In Redux you also could call store.dispatch({ type: 'MyAction' }) directly from the store you've created and exported from store.js. Not only via useDispatch / connectDispatchToProps

Reply:
I'll update this a bit and will make as simple separate effect could be re-used for example for GDStore if it would also need in the future
because now it's looking more like workaroud